scripts/pugme.js

```diff
@@ -0,0 +1,28 @@
+// Description:
+//   Pugme is the most important thing in life
+//
+// Dependencies:
+//   None
+//
+// Configuration:
+//   None
+//
+// Commands:
+//   randbot pug me - Receive a pug
+//   randbot pug bomb N - get N pugs (max 10)
+
+module.exports = function (robot) {
+  robot.respond(/pug me/i, (msg) => {
+    msg.http('http://pugme.herokuapp.com/random').get()((err, res, body) => {
+      msg.send(JSON.parse(body).pug);
+    });
+  });
+  robot.respond(/pug bomb( (\d+))?/i, (msg) => {
+    console.log(msg.match[2]);
+    const count = parseInt(msg.match[2], 10) > 10 ? 10 : msg.match[2] || 5;
+
+    return msg.http(`http://pugme.herokuapp.com/bomb?count=${count}`).get()((err, res, body) => {
```

Comment:
We shouldn't need to return the result of `msg.http().get()` here. Gonna have to promisify these `get` calls somehow...hmmm.

Reply:
Personally, I’d say let’s do a mvp that takes away the ability for anyone to crash our slack or ec2 instance and the rest can be follow up

Reply:
Oh yeah, promisifying is not something that we should introduce now. I created an issue for it so this comment is non-blocking. Sorry, should have said that.